flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml

```diff
@@ -223,6 +223,10 @@ under the License.
 									<pattern>org.apache.parquet</pattern>
 									<shadedPattern>org.apache.hive.shaded.parquet</shadedPattern>
 								</relocation>
+								<relocation>
+									<pattern>shaded.parquet</pattern>
+									<shadedPattern>org.apache.flink.hive.reshaded.parquet</shadedPattern>
```

Comment:
I'm thinking whether it's a good shade pattern in there. it's same to the shade pattern in connector-hive jar. It may overwride the classes in connector-hive jar.
How about reshading to `org.apache.hive.reshaded.parquet` like we don in FLINK-23074?

Reply:
@luoyuxia  Thanks for the reply. I've understood that the pattern should be different from ```connector-hive jar```. 
```org.apache.hive.reshaded.parquet``` looks good to me.

However I have one more question about this below part https://github.com/apache/flink/blob/e01183c8da082b1586dd4eba6ec24f29e6d2fe12/flink-connectors/flink-sql-connector-hive-2.3.9/pom.xml#L114

Don't we need to also change ```flink-sql-connector-hive-2.3.9```, to avoid overwriding classes from ```connector-hive.jar```? 
This part seems to be changed from this [pr](https://github.com/apache/flink/pull/16012), I'm wondering if you are aware of this change before?

Reply:
Haven't been aware of that before. But from the [comment](https://github.com/apache/flink/pull/16012#issuecomment-919990782), seems we also should change it.

Reply:
@luoyuxia I've applied your review. Thanks for the help, PTAL